### ISSUE TITLE
update button reverts to random hash url

### ISSUE THREAD

**Issue by mthomas7** [XimeraProject/ximeraLatex#49]
When a change occurs and the update button is clicked, it reverts back to the randomly generated url instead of refreshing the url currently being used.

**Comment by kisonecat** [comment]
The nice URL is pointing to the "newest version for which the learner has
some data" while that hash URL points to a specific version of the
activity.  So I'm using this basically as a hack to bring up a specific
version.

I could use `history.pushState` to fix this?  I really would like to
understand better what the "right" thing to do here is.

**Comment by oguzkurt** [comment]
Yep, this is an issue of mine too. It is very confusing to the students!
